stix/idl/processing/spectrogram/stx_science_data_lightcurve.pro

```diff
@@ -40,6 +40,10 @@
 ;    pix_ind : in, type="int array", default="all pixels present in observation"
 ;               indices of pixels to sum when making spectrogram - N.B. only applicable when input file is pixel data
 ;
+;    time_range : in, type="dbl array" or "str array", default="all times present in observation"
```

Comment:
If the is a double what is it `ut` time, maybe state here?

Reply:
I've created a new commit and updated the doc string of `time_range`